app/views/buyers/accounts/edit.html.slim

```diff
@@ -3,15 +3,14 @@
 
 - content_for :page_header_title, "Edit account #{@account.org_name} details"
 
-div
-  = semantic_form_for @account,
-                      url: admin_buyers_account_path(@account),
-                      builder: Fields::FormBuilder  do |form|
-    = form.inputs name: 'Contact Details', class: 'inputs' do
+div class="pf-c-card"
+  div class="pf-c-card__body"
+    = semantic_form_for @account, builder: Fields::PatternflyFormBuilder,
+                        url: admin_buyers_account_path(@account),
+                        html: { class: 'pf-c-form pf-m-limit-width' } do |form|
       = form.user_defined_form
 
-    / TODO: remove div and move class to form element
-    div class='pf-c-form'
       = form.actions do
         button class="pf-c-button pf-m-primary" type="submit" Update Account
+        a class="pf-c-button pf-m-link" href=admin_buyers_account_path(@account) Cancel
```

Comment:
```suggestion
        a class="pf-c-button pf-m-secondary" href=admin_buyers_account_path(@account) Cancel
```
WDYT about using `secondary` class so it looks more buttonish than it does now?

Reply:
> WDYT about using `secondary` class so it looks more buttonish than it does now?

I think this is against PF4 guidelines:

https://v4-archive.patternfly.org/v4/components/button/design-guidelines#usage

> The Cancel button should always be a [link button](https://v4-archive.patternfly.org/v4/components/button/design-guidelines#link-button).


And it's like that on all other forms.

So, I just propose to swap Delete and Cancel.

Reply:
I confirm making Cancel a secondary is a no go. Also having a Delete button is against the rules, but we lack of a proper design for it. What I suggest is aligning Delete to the right end, moving it out of the "form buttons" group:
![Screenshot 2023-11-21 at 13 38 28](https://github.com/3scale/porta/assets/11672286/e656f1fc-dd9a-47b9-b251-674477136b01)

Reply:
Alternatively, we can add a "Danger zone" form group with a link to delete de account.

Reply:
I like the idea of adding a danger zone in the show page, it makes sense to me.

Reply:
OK that's a substantial change and will require UX and probably Andreu. It also will affect all other delete buttons in edit pages. We can bring it to then next time. In the meantime I suggest we merge this and carry on.

Reply:
@josemigallas is there a problem in having it like this?

![image](https://github.com/3scale/porta/assets/1270328/133367e6-1255-4cd9-8bb1-3bee5af308c5)

To me a Cancel link button separating two "normal" buttons looks really weird :shrug:

Reply:
OK I am going to revert the cancel button and leave the minimal change. This PR is only about Patternfly. I [created a JIRA](https://issues.redhat.com/browse/THREESCALE-10517) for UX to work on an actual new design of this "danger zone" and additionally we can add it to the tables as a row action.

That said, I think we can merge this and move on. Thanks for your comments and suggestions!!